### PR TITLE
Can load a profile with an email adress and view users profiles

### PR DESCRIPTION
**Pull request by MailyLehoux**
The idea is to load a profile URL, in order to allow the current user to check other user's profile page. Futhermore, we have to carry out the 2 types of views : If we check an other pofile, we are not autorized to change the sections. 

At the moment, we can view a profile like this : "profile/[email]" , it was the easier way to test the loading of profile page cause its an unique key.  

But we can imagine to load this URL like this as well : "profile/[name]/[uniqueID]", and uniqueID could be an unique UUID. 

I don't know if we have other possibilties. What do you think about that ?

### CODE REVIEW

**Comment by Ked57** [comment]
There's no uuid for users, email is used as a PK in the database. So email is the right way for sure 😄 And users will understand what profile they're going to visit before clicking on the link so I don't see any downside right now

File: hasura/migrations/1634121710190_alter_table_public_User_drop_column_id/up.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+alter table "public"."User" drop column "id" cascade;
```
Comment (Ked57) [inline hasura/migrations/1634121710190_alter_table_public_User_drop_column_id/up.sql:1]: Seems like all of the migrations shouldn't have been commited

File: questions.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+
+
+
+1- 
+
+
+
+-- objetcifs
+créer une nouvelle route profil/{userName}
```
Comment (Ked57) [inline questions.txt:17]: Same for this

File: src/components/SidePanel.tsx [hunk 3]
```diff
@@ -7,6 +7,7 @@ import { LocaleSelector } from "./LocaleSelector";
 import { DarkModeSelector } from "./DarkModeSelector";
 import { useDarkMode } from "../utils/darkMode";
 import { useRouter } from "next/router";
+import { userInfo } from "os";
```
Comment (Ked57) [inline src/components/SidePanel.tsx:10]: Seems like VSCode imported this for no reason

**Comment by MailyLehoux** [comment]
This PR seems done for me. Waiting for your reviews to close it :100: 

I've also changed the order of the diferents profile page components refered by Stéphane's Figma : https://www.figma.com/proto/0kOnLSj4J9FPcAvlRyupGI/skillZ?node-id=231%3A1039&starting-point-node-id=107%3A0

The next step, in order of this PR, could be the implementation of an historic of the routing, what do you think about that ?

**Pull request merged**